### DISCUSSION TITLE
Change document signing certificate

### DISCUSSION THREAD

**Discussion by BenoitLama** [OpenSignLabs/OpenSign#1193] [Q&A]
Hello

We would like to change the self-signed certificate for signing documents so that it is also valid in Adobe.
Has anyone already changed this certificate? If so, by what type of certificate?

Thank you for your feedback

**Comment by andrew-opensignlabs** [comment]
@BenoitLama You can change the certificate on self-hosted version through environment variables. You will need a document signing certificate from an [adobe accredited CA](https://helpx.adobe.com/in/acrobat/kb/approved-trust-list1.html).
